docs/docs/overview/quickstart/get-started-with-React.md

```diff
@@ -0,0 +1,230 @@
+
+   ![Configuring the In-app channel](https://res.cloudinary.com/dxc6bnman/image/upload/v1685465725/guides/in-app-configuration_1_qzwd93.png)
+
+I'll briefly explain the function of each label in the image above:
+
+- **1-Preview**: Shows you a glimpse of how each notification item will look like in the Notification Center UI.
+- **2-Avatar**: If turned on, each notification item will show the avatar of the subscriber.
+- **3-Action**: With this, you can add a primary and secondary call to action button to each notification item.
+- **4-Notification Feeds**: This displays a stream of specific notifications. You can have multiple feeds to show specific notifications in multiple tabs.
+- **5-Redirect URL**: This is the URL to which a subscriber can be directed when they click on a notification item.
+- **6-Filter**: This feature allows you to configure the criteria for delivering notifications. For instance, you can apply a filter based on a subscriber's online status to send them an email if they were online within the last hour. Read [more about filters](https://docs.novu.co/platform/step-filter/#subscriber-seen--read-filters).
+- **7-Editor**: You can add text that you want displayed in each notification item. Additionally, you can specify custom variables using `{{ }}`. This means you can inject variables from your code into a notification item's text via a payload.
+
+In our case, we'll use the custom variables functionality, as shown below:
+
+![Using custom variables in Novu](https://res.cloudinary.com/dxc6bnman/image/upload/v1686742931/guides/Screenshot_2023-05-28_at_3.36.10_AM_o6oosa.png)
+
+Feel free to add only text for now and rename the notification template to `quickstart`. It automatically creates a slug-like Identifier that will be needed in later steps to trigger a notification.
+
+![Creating a notification template from Novu dashboard](https://res.cloudinary.com/dxc6bnman/image/upload/v1686743094/guides/Screenshot_2023-05-28_at_3.40.01_AM_ulmqvm.png)
+
+Now, we’ll learn how to create subscribers on Novu - Recipients of Notifications!
+
+## Create a subscriber
+
+If you click “Subscriber” on the left sidebar of [Novu dashboard](https://web.novu.co/subscribers), you’ll see the subscriber list. By default, there will only be one subscriber as you’re automatically added as a subscriber when you sign up for Novu:
+
+![Subscribers page on the Novu dashboard](https://res.cloudinary.com/dxc6bnman/image/upload/v1685465978/guides/subscribers_1_opmw0s.png)
+
+Now, let’s create a subscriber on Novu. After creating a subscriber, we’ll trigger notification to this subscriber. Subscribers are identified by a unique `subscriberId`.
+
+With Novu, you can create a subscriber using any of its SDKs (Node.js, PHP, .NET, Go, Ruby, Python and Kotlin). The code to create a subscriber in Novu is:
+
+```jsx
```

Comment:
Starting from here, all the node code examples should show in both Node, and PHP. 

Check out the code sample here https://docs.novu.co/platform/preferences/#get-subscriber-preferences